src/Widgets/ConvertPDF.vala

```diff
@@ -91,8 +91,9 @@ namespace pdftricks {
             grid.orientation = Gtk.Orientation.VERTICAL;
             grid.halign = Gtk.Align.CENTER;
             grid.valign = Gtk.Align.CENTER;
-            grid.column_spacing = 12;
-            grid.row_spacing = 6;
+            grid.column_spacing = 16;
+            grid.row_spacing = 8;
+            grid.set_column_homogeneous(true);
```

Comment:
```suggestion
            grid.column_homogeneous = true;
```

elementary coding styles prefer using properties instead of `set_*` methods if possible. More info: https://elementary.io/docs/code/reference#equals-in-place-of-set